web/html/src/components/input/Label.js

```diff
@@ -17,7 +17,6 @@ function Label(props: Props) {
     >
       { props.name }
       { props.required ? <span className="required-form-field"> *</span> : undefined }
-            :
```

Comment:
I will revert it, I see all the old code is using that ":"

IMO It looks quite ugly and useless, but even if we remove them it should in another PR

![Screenshot from 2019-03-12 00-24-26](https://user-images.githubusercontent.com/1140720/54166507-6e9de680-445d-11e9-93ac-7d9846d35a64.png)
 cc @ncounter

Reply:
Ah, I see. Yeah it is quite ugly in the most of the cases, but sometimes we don't have the `input` field right after, but a simple following text and the `column` is kind of needed to split the two parts. It could be definitely designed better, but it is too much spread around to softly remove it and *see what happens after*. Let's keep it and we will tackle this later on a UI/design round. :wink: